roles/cobbler/defaults/main.yml

```diff
@@ -50,3 +50,8 @@ ansible_user_uid_: ''
 #      - "nameserver x.x.x.x"
 #      - "search an.example.com"
 resolvconf: []
+
+# set to true if /etc/fstab must be modified to persist things like
+# mount options, which is useful for long lived bare metal machines,
+# less useful for virtual machines that are re-imaged before each job
+modify_fstab: true
```

Comment:
I just noticed this, sorry.  'modify_fstab' is used in the testnode role, so this should live in roles/testnode/defaults/main.yml